verifier/src/lib.rs

```diff
@@ -67,4 +69,72 @@ mod tests {
         let verifier = WormholeVerifier::new();
         verifier.verify(proof).unwrap();
     }
+
+    #[test]
+    fn cannot_verify_with_modified_exit_account() {
+        let prover = WormholeProver::new();
+        let inputs = CircuitInputs::default();
+        let mut proof = prover.commit(&inputs).unwrap().prove().unwrap();
+
+        println!("proof before: {:?}", proof.public_inputs);
+        let exit_account = ExitAccount::from_field_elements(&proof.public_inputs[15..19]);
```

Comment:
Actually, let's also quickly make the `ExitAccount`'s indices a constant, `public_inputs` is deterministic but any time we add or remove any the indices might change